frontend/app/src/components/display/StatCardWide.vue

```diff
@@ -0,0 +1,93 @@
+<template>
+  <v-card class="stat-card-wide" :loading="!locked && loading">
+    <v-row v-if="!locked" no-gutters>
+      <v-col
+        class="stat-card-wide__first-col pa-6"
+        cols="12"
+        :sm="colsSize[cols]"
+      >
+        <slot name="first-col"></slot>
+      </v-col>
+      <v-col
+        class="stat-card-wide__second-col d-flex"
+        :class="{
+          'stat-card-wide__second-col--horizontal': $vuetify.breakpoint.smAndUp
+        }"
+        cols="12"
+        :sm="colsSize[cols]"
+      >
+        <v-divider :vertical="$vuetify.breakpoint.smAndUp"></v-divider>
+        <div class="stat-card-wide__second-col__content pa-6 flex-grow-1">
+          <slot name="second-col"></slot>
+        </div>
+        <v-divider
+          v-if="cols !== 'two'"
+          :vertical="$vuetify.breakpoint.smAndUp"
+        ></v-divider>
+      </v-col>
+      <v-col
+        v-if="cols !== 'two'"
+        class="stat-card-wide__third-col pa-6"
+        cols="12"
+        :sm="colsSize[cols]"
+      >
+        <slot name="third-col"></slot>
+      </v-col>
+      <v-col
+        v-if="cols !== 'two' && cols !== 'three'"
```

Comment:
Why not use numerical mapping instead? this woudl make the checks easier.

Reply:
Good idea.